tests/test_conf.py

```diff
@@ -23,6 +24,14 @@
 target-version = ['py36', 'py37', 'py38']
 """
 
+JSON_CONFIG = {
```

Comment:
For this test case, I'd suggest using raw string instead of `dict` which might be a better simulation to our real use case.

e.g.,

```python
"""
    "commitizen": {
        "name": "cz_jira",
        "version": "1.0.0",
        "version_files": ["commitizen/__version__.py", "pyproject.toml"],
        "style": [["pointer", "reverse"], ["question", "underline"]],
    }
"""
```